.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": false
```

Comment:
`semi`, `tabWidth` and `useTabs` are all set to their defaults so can be removed

Hot take: `singleQuote` should be `false` (which is the default so removed), and `trailingComma` should be `none`